app/build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
 }
 
 dependencies {
+    implementation 'io.quarkus:quarkus-container-image-docker'
```

Comment:
Why not move it above the current line 15.
It looks nicer with the other Quarkus implementations.
And the BOM is then always the first one.